turf/invariant/_invariant.py

```diff
@@ -173,6 +180,110 @@ def get_geometry_from_features(features: Any, allowed_types: Sequence = None) ->
     raise InvalidInput(error_code_messages["InvalidGeometry"](allowed_types))
 
 
+def get_geometry_type(features: Any, allowed_types: Sequence = None) -> Tuple[str]:
+    """
+    Gets the Geometry type from Features. Features must be a GeoJSON,
+    a Feature object, FeatureCollection a Dictionary or a List, otherwise it raises an exception.
+
+    For Coordinate list, please be aware that the whole feature has to wrapped in a list.
+
+    :param features: Any input value(s)
+    :param allowed_types: allowed Feature types
+    :return: tuple with extracted geometry types
+    """
+    if not allowed_types:
+        allowed_features = [allowed_features_default, allowed_types_default]
+
+    else:
+
+        allowed_features = [
+            tuple(
+                [
+                    feature
+                    for feature in allowed_features_default
+                    if feature.__name__ in allowed_types
+                ]
+            ),
+            allowed_types,
+        ]
+
+    features = get_geometry_from_features(features, allowed_features[1])
+
+    if isinstance(features, (dict, *allowed_features[0])):
+
+        geometry_type = _get_geometry_type_from_feature(features, allowed_features)
+
+    elif isinstance(features, (list, tuple)):
+
+        geometry_type = _get_geometry_type_from_list(features, allowed_features)
+
+    else:
+        raise InvalidInput(error_code_messages["InvalidGeometry"](allowed_features[1]))
+
+    return geometry_type
```

Comment:
I would add here before returning some logic for returning a string if the tuple has only one element, so something like this:

```python
if len(geometry_type) == 1:
    geometry_type = geometry_type[0]
```

Reply:
this makes the return more precise. thanks.